scalafmt-cli/src/main/scala/org/scalafmt/cli/CliOptions.scala

```diff
@@ -92,13 +92,42 @@ case class CliOptions(
     gitOpsConstructor: AbsoluteFile => GitOps = x => new GitOpsImpl(x),
     noStdErr: Boolean = false
 ) {
+  private[this] val DefaultGit = false
+  private[this] val DefaultFatalWarnings = false
+  private[this] val DefaultIgnoreWarnings = false
+  private[this] val DefaultEncoding = Codec.UTF8
```

Comment:
These default values are borrowed from here https://github.com/scalameta/scalafmt/blob/f2154330afa0bc4a0a556598adeb116eafecb8e3/scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala#L160

Though it is not reasonable to manage default values in both `scalafmt-cli` and `scalafmt-core`, we need to refer those configuration values from cli side...